movie-blog/src/App.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import CreateElement from "./components/create-element";
+import ComponentElement from "./components/component";
+import PureComponentElement from "./components/pure-component";
+import FunctionalComponentElement from "./components/functional-component";
+
+const createElement = React.createElement(
```

Comment:
You don't seem to be using this particular variable anywhere in your codebase, so you may delete that.